pkg/controller/statusmanager/status_manager.go

```diff
@@ -212,23 +214,58 @@ func (status *StatusManager) SetNotFailing(level StatusLevel) {
 	status.syncFailing()
 }
 
+func (status *StatusManager) SetAttributes(daemonSets []types.NamespacedName, deployments []types.NamespacedName, containers []opv1alpha1.Container) {
+	status.mux.Lock()
+	defer status.mux.Unlock()
+	status.daemonSets = daemonSets
+	status.deployments = deployments
+	status.containers = containers
+}
+
 func (status *StatusManager) SetDaemonSets(daemonSets []types.NamespacedName) {
```

Comment:
We don't need these setters anymore. The only use of them was replaced with SetAttributes. Having one attomic setter and couple of getters looks fine to me.

Reply:
Removed